File: src/main/java/org/jukeboxmc/potion/JumpBoostEffect.java
package org.jukeboxmc.potion;

import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class JumpBoostEffect extends Effect {

    @Override
    public int getId() {
        return 8;
    }

    @Override
    public EffectType getEffectType() {
        return EffectType.JUMP_BOOST;
    }

    @Override
    public Color getEffectColor() {
        return new Color( 34, 255, 76 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/JumpBoostEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class JumpBoostEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 8
    }

    override fun getEffectType(): EffectType {
        return EffectType.JUMP_BOOST
    }

    override fun getEffectColor(): Color {
        return Color(34, 255, 76)
    }

    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {

    }
}